UNITTESTS/README.md

```diff
@@ -0,0 +1,65 @@
+## Stubs, mocks and fakes in Mbed OS
+
+To make unit testing easier, Mbed OS provides some ready-made CMake libraries you can add in your `target_link_libraries`. 
+Libraries can be object files that contain stubs, mocks and fakes but also sets of include paths - this way you can gain access to normal headers from Mbed OS.
+
+All the libraries available are in the `CMakeLists.txt` files in `mbed-os/UNITTESTS/stubs` and `mbed-os/UNITTESTS/fakes`.
+
+The most common ones you might need are `mbed-os-fakes-ble` and `mbed-os-fakes-event-queue`.
```

Comment:
```suggestion
```

Shall we just remove this, I would add just that libraries are prefixed with `mbed-os-fakes` in the next section.